addons/binding/org.openhab.binding.max.test/pom.xml

```diff
@@ -0,0 +1,115 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+		 <groupId>org.openhab.binding</groupId>
+	<artifactId>org.openhab.binding.max.test</artifactId>
+	<packaging>eclipse-test-plugin</packaging>
+	<name>openHAB MAX! Binding Tests</name>
```

Comment:
like the bindings, the name should be without "openHAB" here, i.e. simply "MAX! Binding Tests"